packages/pigeon/lib/generator_tools.dart

```diff
@@ -624,3 +622,9 @@ class OutputFileOptions<T> {
   /// Options for specified language across all file types.
   T languageOptions;
 }
+
+/// Provides a trailing comma for strings that are greater than 75 len
```

Comment:
Isn't this specific to Dart, so something that would make more sense in the Dart generator?